__tests__/keyboard.test.tsx

```diff
@@ -0,0 +1,39 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { JSXElementConstructor, ReactElement } from 'react';
+import KeyboardPage from '../pages/[keyboard]';
+import { getLanguageContext } from '../utils/context';
+
+const setup = (tsx: ReactElement<any, string | JSXElementConstructor<any>>) => {
+  return {
+    user: userEvent.setup(),
+    ...render(tsx),
+  };
+};
+
+describe('Japanese keyboard', () => {
+  const props = getLanguageContext('japanese');
+
+  const convert = [
+    ['nwarayamahanatasakaapabadazaga', 'んわらやまはんあたさかあぱばだざが'],
+    ['rimihinichishikiipibidijigi', 'りみひんいちしきいぴびぢじぎ'],
```

Comment:
This will allow us to more easily test the text conversion

Reply:
Cool